dlfcn.c

```diff
@@ -474,6 +474,48 @@ char *dlerror( void )
     return error_buffer;
 }
 
+int dladdr (const void *addr, Dl_info *info)
+{
+   // only returns filename, FWIW.
+   TCHAR  tpath[PATH_MAX];

```

Comment:
`TCHAR` type is unsafe as it may be expanded to `wchar_t` which is incompatible with types in `struct dl_info`.